.storybook/config.js

```diff
@@ -39,13 +39,16 @@ const prefix = (phrase, prefix) => value => (value.indexOf(phrase) > -1 ? prefix
 const pipe = (...fns) => value => fns.reduce((result, fn) => fn(result), value);
 
 function storySort(a, b) {
+  // console.warn('b', b);
   const prefixFn = pipe(
     prefix('welcome-', '0'),
     prefix('getting-started', '0'),
     prefix('tokens-', '1'),
-    prefix('components-', '2'),
+    prefix('components-', '3'),
+    prefix('states', '4'),
```

Comment:
This works, but I think we may want to do this separately from this calculation. This is specifically meant to target the prefix from the categories at the top level of storybook. I believe this could result in some unexpected edge cases for something like a story named `Tokens|Foo/Bar/States`. I would suggest doing a separate check after the number prefix has already been added.

Reply:
This function replaced another function that only handled top-level categories. The input of that function was something like `['Welcome', 'Tokens', 'Components']`, but I also needed to make sure "Getting Started" was the first item under "Welcome" so that Storybook picked it as the default page. Numbers were chosen because sorting alphabetically ensures numbers rank higher than letters, so '0first' will come before 'first'. Also numbers are easy for us to understand in terms of "ranking".

This function is definitely a hack, but the intent is to be powerful enough to handle sorting on a global level while still being easy enough to add new sorting rules. I actually think adding `prefix('states', '3')` _before_ the `prefix('components-', '3')` would accomplish the same goal without introducing a new sorting hierarchy ('3' would be prefixed for having 'states' in the storyname first, then '3' would be prefixed first for being in the "Components" category).

Reply:
Actually, I retract the `3`. It has to be a `4` to not be the first story in `Components`... Perhaps not very intuitive